docs/guides/Redux-Integration.md

```diff
@@ -44,3 +44,13 @@ class App extends React.Component {
 Now, your navigation state is stored with redux, and you can fire navigation actions using redux.
 
 When a navigator is given a `navigation` prop, it relinquishes control of the state. So you are now responsible for persisting state, handling deep linking, integrating the back button, etc.
+
+When you are nesting navigators, the navigation state is automatically passed down as long as a navigator is attached as a screen. For example:
```

Comment:
It is a bit hard for me to understand when I read this sentence, I've read it a few times to get what you mean.

I would phrase it something like:

The navigation state is passed down from one navigator to another when you nest them. The child navigator, in order to receive the state from its parent, needs to be defined as a `screen`.